lib/compute/jenkins-main-node.ts

```diff
@@ -393,6 +393,12 @@ export class JenkinsMainNode {
                   auto_removal: true,
                   log_stream_name: 'workflow-logs',
                 },
+                {
+                  file_path: '/var/lib/jenkins/jobs/gradle-check/builds/*/log',
+                  log_group_name: 'JenkinsMainNode/gradle-check.log',
+                  auto_removal: true,
+                  log_stream_name: 'gradle-check.log/{date}',
```

Comment:
Would date create too many log-streams? Also forgot what auto-removal does. Any idea?

Reply:
There is no limit on how many log streams can be created. 
The choice is either push everything into one log stream or have one created daily so that it becomes easier to find the log one is looking for and faster, since it will contain only limited logs.

Reply:
Reg `auto_removal`, it is mainly helpful when you have some sort of log rotation happening, particularly on logs that are suffixed, e.g., sys.log is your main log file and your rotation logs creates sys.log.1 after each hour. In this case CW is supposed to clean up the rotated log. 

But in our case it is not the case and the flag doesn't do anything.

Reply:
Yeah we can think about adding log retention policy. I believe it will be just too much logs. Even jenkins has 30 day retention policy for gradle-check logs. Can come as an enhancement.